zest/releaser/pypi.py

```diff
@@ -277,12 +277,26 @@ def create_wheel(self):
         [zest.releaser]
         create-wheel = yes
 
+        If there is no setting for ``create-wheel``, then if universal
+        wheels are specified as in the following example, it is treated as if
+        ``create-wheel`` was true:
+
+        [bdist_wheel]
+        universal = 1
+
         """
         if not USE_WHEEL:
             # If the wheel package is not available, we obviously
             # cannot create wheels.
             return False
-        return self._get_boolean('zest.releaser', 'create-wheel')
+        create_setting = self._get_boolean('zest.releaser', 'create-wheel', self)
+        if create_setting is not self:
```

Comment:
The comparison to `self` looks really weird to me. I realise what you are doing, but can you change this into comparison with `None` instead?